nbhttp/engine.go

```diff
@@ -695,9 +711,9 @@ func NewEngine(conf Config) *Engine {
 			c.DataHandler(c, data)
 		}
 	})
-	g.OnWriteBufferRelease(func(c *nbio.Conn, buffer []byte) {
-		mempool.Free(buffer)
-	})
+	// g.OnWriteBufferRelease(func(c *nbio.Conn, buffer []byte) {
```

Comment:
did you intend to leave this comment out?

Reply:
will clean it